dev/grouped-table.vue

```diff
@@ -140,7 +146,36 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.row-style{
-    background-color: red;
+.row-style {
+  background-color: red;
+}
+
+#vgt-root >>> {
```

Comment:
I tried applying a little example styling. But it won't work.
I'm not used to using scss, so maybe I foobar'd there?